exercises/kindergarten-garden/src/example/java/KindergartenGarden.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+
+
+public class KindergartenGarden {
+    private String[] students;
+    private String garden;
+    private final int PLANTS_PER_STUDENT_PER_ROW = 2;
```

Comment:
Normally this formatting, uppercase & [snake_case](https://en.wikipedia.org/wiki/Snake_case), is only used for `final static` variables. Normal [camelCase](https://en.wikipedia.org/wiki/Camel_case) is used for `final` variables :)